src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -739,35 +739,18 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                             $maxDepth = 10
                             $ellipsis = ""`u{2026}""
                             $resetColor = ''
-                            if ($Host.UI.SupportsVirtualTerminal) {
-                                $resetColor = ""`e[0m""
+                            if ($Host.UI.SupportsVirtualTerminal -and !(Test-Path env:__SuppressAnsiEscapeSequences)) {
+                                $resetColor = [System.Management.Automation.VTUtility]::GetEscapeSequence(
+                                    [System.Management.Automation.VTUtility+VT]::Reset
+                                )
                             }
 
-                            function Get-VT100Color([string] $color) {
-                                if (! $Host.UI.SupportsVirtualTerminal) {
+                            function Get-VT100Color([ConsoleColor] $color) {
+                                if (!$Host.UI.SupportsVirtualTerminal -or (Test-Path env:__SuppressAnsiEscapeSequences)) {
```

Comment:
Can you set this variable in the linux and mac YAML

Reply:
will do

Reply:
Also adding to Windows unless there is a reason it works there